lib/player.rb

```diff
@@ -0,0 +1,34 @@
+class Player
```

Comment:
Ideally, we should separate classes into their own separate files as it scales better and is easier for others to navigate the codebase. If this project were to get more complex, files like this can grow quite large.

Instead of a flat project structure, we can introduce a `players` directory and create a `Players` module.

```ruby
# lib/players/player.rb
module Players
  class Player
  end
end

# lib/players/human.rb
module Players
  class Human
  end
end

# lib/players/computer.rb
module Players
  class Computer
  end
end
```